### ISSUE TITLE
[OPKGC] the rel tag is not correctly handled

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#590]
**Reported by valleegr on 22 Apr 2010 21:47 UTC**
Discussion between Jean and Olivier on the mailing list:

```
hi,

I've fixed the file and tested using
cd ~oscar_svn/packages
opkgc --dist=rhel --input=ganglia
The generated rpms don't seem to have a Requires: xxxx >= 3.0.6 tag.

anyway, in case it helps, I've attached a patch.

Regards,

Olivier.

> Hi,
> The pkg tag has a specific attribute for specifying the relation with
> the version. AFAIK, it is "rel", which would give:
> 
> <pkg version="3.0.6" rel=">=" >ganglia-mod</pkg>
> 
> Regards,
> Jean
```

**Comment by dikim33** [comment]
**Comment by valleegr on 6 Oct 2010 19:53 UTC**
I just tested with opkgc-0.4.5 on Debian based systems and it is working just fine, the dependencies are correct.

**Comment by dikim33** [comment]
**Comment by valleegr on 6 Oct 2010 20:43 UTC**
I just tested on CentOS-5 i386 with opkgc-0.4.5 and i confirm the problem: the version requirement for a given package is not included, instead the version number is included as a required package, which is obviously very bad. I upgrade the bug to blocker since we cannot release a decent version of OSCAR for RPM based systems because of that bug.

**Comment by dikim33** [comment]
**Comment by valleegr on 3 Jan 2011 10:44 UTC**
(In [9314]) This check-in fixes #590 - Correctly deal with the "rel" tag of config.xml files on RPM 
based systems.
